cli/inputhandler/options.cpp

```diff
@@ -0,0 +1,64 @@
+#include "options.h"
+
+CLIOptions::CLIOptions() :
+        m_optionsTx("transaction", "Manage transactions"),
+        m_optionsPem("pem", "Manage pem files")
+{
+    initOptions();
+}
+
+cxxopts::Options CLIOptions::transaction() const
+{
+    return m_optionsTx;
+}
+
+cxxopts::Options CLIOptions::pem() const
+{
+    return m_optionsPem;
+}
+
+std::string CLIOptions::help() const
+{
+    return helpTx() + "\n" + helpPem();
+}
+
+std::string CLIOptions::helpPem() const
+{
+    return m_optionsPem.help();
+}
+
+std::string CLIOptions::helpTx() const
+{
+    return m_optionsTx.help();
+}
+
+void CLIOptions::initOptions()
+{
+    initOptionsTx();
+    initOptionsPem();
+}
+
+void CLIOptions::initOptionsTx()
+{
+    m_optionsTx.add_options("new") // transaction new group
+            ("nonce", "Transaction nonce", cxxopts::value<uint64_t>())
+            ("value", "Transaction value", cxxopts::value<std::string>())
+            ("receiver", "Receiver's address", cxxopts::value<std::string>())
+            ("receiver-username", "Receiver's username", cxxopts::value<std::string>()->default_value(""))
+            ("sender-username", "Sender's username", cxxopts::value<std::string>()->default_value(""))
+            ("gas-price", "Transaction gas price", cxxopts::value<uint64_t>())
+            ("gas-limit", "Transaction gas limit", cxxopts::value<uint64_t>())
+            ("data", "Transaction data/payload", cxxopts::value<std::string>()->default_value(""))
+            ("chainID", "Chain identifier", cxxopts::value<std::string>()->default_value("T"))
+            ("version", "Transaction version", cxxopts::value<uint64_t>()->default_value("1"))
+            ("options", "Transaction options", cxxopts::value<uint32_t>()->default_value("0"))
+            ("pem", "Pem keyfile", cxxopts::value<std::string>())
+            ("outfile", "Json file where the output will be stored", cxxopts::value<std::string>());
+}
+
+void CLIOptions::initOptionsPem()
+{
+    m_optionsPem.add_options("load") // pem load group
```

Comment:
Not sure what is the purpose of this command.

Reply:
Actually, you are right.
One of the first requirements of this SDK was to load a pem file to check if it was valid. I kept carrying this implementation, but right now it just does **not** make sense to keep it anymore.
When you try to create a new transaction or issue/transfer ESDTs, we check the validity of the pem file anyway.

I will **DELETE** it